ios/build/Build/Intermediates.noindex/XCBuildData/PIFCache/target/TARGET@v8_hash=8A7FE9388A27CE2830CF91823B64AE8F-json

```diff
@@ -0,0 +1 @@
e91d3be0bac269ea","guid":"d1b7560c4abbdd3aebce48be11a78dd2d75bf9cbcff7cf8f4ab20a0e0087687b","intentsCodegenFiles":"true"},{"fileReference":"d1b7560c4abbdd3aebce48be11a78dd2f2452297126207d8e60d98d10be6652b","guid":"d1b7560c4abbdd3aebce48be11a78dd2d95f768d569468fb23dbf0a73d00de3e","intentsCodegenFiles":"true"},{"fileReference":"d1b7560c4abbdd3aebce48be11a78dd213cdc65b66223163b775eb6123594652","guid":"d1b7560c4abbdd3aebce48be11a78dd25815a58da0ec28b1d6757f4e7d9ea845","intentsCodegenFiles":"true"}],"guid":"d1b7560c4abbdd3aebce48be11a78dd275b4de00c79a55799735b6783ff3c9c7","type":"com.apple.buildphase.resources"},{"buildFiles":[],"emitEnvironment":"false","guid":"d1b7560c4abbdd3aebce48be11a78dd2796fb2b379861a6f8fdb52a44826240e","inputFileListPaths":[],"inputFilePaths":[],"name":"[CP] Embed Pods Frameworks","originalObjectID":"8D34C7FE3D99F20E5EA1B18A","outputFileListPaths":[],"outputFilePaths":[],"scriptContents":"\"${SRCROOT}/Pods/Target Support Files/Pods-movement-vites/Pods-movement-vites-frameworks.sh\"\n","shellPath":"/bin/sh","type":"com.apple.buildphase.shell-script"},{"buildFiles":[],"emitEnvironment":"false","guid":"d1b7560c4abbdd3aebce48be11a78dd2276ecc63bc48704b9d6fa31456054c62","inputFileListPaths":[],"inputFilePaths":[],"name":"[CP] Copy Pods Resources","originalObjectID":"AA38385B7D8D0854176B72FC","outputFileListPaths":[],"outputFilePaths":[],"scriptContents":"\"${SRCROOT}/Pods/Target Support Files/Pods-movement-vites/Pods-movement-vites-resources.sh\"\n","shellPath":"/bin/sh","type":"com.apple.buildphase.shell-script"},{"buildFiles":[],"emitEnvironment":"true","guid":"d1b7560c4abbdd3aebce48be11a78dd20c85f99c4752103ea206128fc8f469b8","inputFileListPaths":[],"inputFilePaths":[],"name":"Bundle Expo Assets","originalObjectID":"19D233B7204FDD7D0087ED51","outputFileListPaths":[],"outputFilePaths":[],"scriptContents":"set -eo pipefail\n\nif [ \"$CONFIGURATION\" == \"Debug\" ]; then\n  echo \"Skipping asset bundling in debug mode.\"\n  exit 0\nfi\n\npushd \"${SRCROOT}/..\"\nvalue=\"$(cat ~/.expo/PATH)\"\ndest=\"$CONFIGURATION_BUILD_DIR/$UNLOCALIZED_RESOURCES_FOLDER_PATH\"\nPATH=\"$PATH:$value\" expo bundle-assets --platform ios --dest \"$dest\"\npopd\n","shellPath":"/bin/sh","type":"com.apple.buildphase.shell-script"}],"buildRules":[],"dependencies":[],"guid":"d1b7560c4abbdd3aebce48be11a78dd22ca86a4205524b3d2cb138cb4dc1c3fa","name":"movement-vites","predominantSourceCodeLanguage":"Xcode.SourceCodeLanguage.Objective-C","productReference":{"guid":"d1b7560c4abbdd3aebce48be11a78dd2001e75eee30986c3fd2893d9cc2340fc","name":"movement-vites.app","type":"product"},"productTypeIdentifier":"com.apple.product-type.application","provisioningSourceData":[{"bundleIdentifierFromInfoPlist":"edu.haverford.MovementVitamins","configurationName":"Debug","legacyTeamID":"C8D8QTF339","provisioningStyle":0},{"bundleIdentifierFromInfoPlist":"edu.haverford.MovementVitamins","configurationName":"Release","legacyTeamID":"C8D8QTF339","provisioningStyle":0}],"type":"standard"}
```

Comment:
I can't imagine that this isnt a generated file

Reply:
removed